packages/kamino-sdk/src/rebalance_methods/math_utils.ts

```diff
@@ -0,0 +1,30 @@
+import Decimal from 'decimal.js';
+import { FullBPSDecimal } from '../utils/CreationParameters';
+import { PositionRange } from '../utils';
+
+export function getPriceRangeFromPriceAndDiffBPS(
+  price: Decimal,
+  lowerDiffBPS: Decimal,
+  upperDiffBPS: Decimal
+): PositionRange {
+  let lowerPrice = price.mul(FullBPSDecimal.sub(lowerDiffBPS)).div(FullBPSDecimal);
+  let upperPrice = price.mul(FullBPSDecimal.add(upperDiffBPS)).div(FullBPSDecimal);
+
+  return { lowerPrice, upperPrice };
+}
+
+export function getResetRangeFromPriceAndDiffBPS(
+  price: Decimal,
+  lowerDiffBPS: Decimal,
+  upperDiffBPS: Decimal,
+  resetLowerDiffBPS: Decimal,
+  resetUpperDiffBPS: Decimal
+): PositionRange {
+  let resetLowerPrice = price
+    .mul(FullBPSDecimal.sub(resetLowerDiffBPS.mul(lowerDiffBPS).div(FullBPSDecimal)))
```

Comment:
nit: extract `resetLowerDiffBPS.mul(lowerDiffBPS).div(FullBPSDecimal)` to a separate variable so it's easier to read